src/redux/features/books/bookSlice.js

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { v4 as uuidv4 } from 'uuid';
+
+const initialState = {
+  value: [],
+};
+
+const BookSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    add(state, action) {
+      state.value.push({
+        id: uuidv4(),
+        genre: action.payload.genre,
+        title: action.payload.title,
+        producer: action.payload.producer,
+      });
+    },
+    remove(state, action) {
+      state.value.filter((book) => action.id !== book.id);
+    },
+  },
+});
+
+export const { add, remove } = BookSlice.actions;
```

Comment:
Thank you for the review @Mihndim2020 however the actions are automatically created on this line when using the createSlice method.